src/components/Form/SelectOneMedia.js

```diff
@@ -73,10 +73,18 @@ function SelectOneButton({ value, onChange }) {
     setMediaChoiceModalOpen(false)
     setAddMediaModalOpen(true)
   }
-
   return value?.docId ? (
     <div className="mt-1 inline-flex border border-transparent bg-white rounded-lg shadow-md text-sm font-medium p-2 space-x-1">
-      <DocumentThumbnail.Container docId={value?.docId} />
+      {docType === IMAGE ? (
+        <MediaThumbnail.Image
+          id={value?.docId}
+          type={docType}
```

Comment:
The type here will always be 'image', and likely the size will always be small, so we could set them inside the `MediaThumbnail.Image` component and avoid passing extra props.

Same for `MediaThumbnail.Video`.

Reply:
Yes, I missed that, good catch.